cev_eris.dme

```diff
@@ -2799,6 +2799,7 @@
 #include "zzz_modular_syzygy\storytellers\lookout.dm"
 #include "zzz_modular_syzygy\storytellers\mentor.dm"
 #include "zzzz_modular_occulus\code\datums\autolathe\guns.dm"
+#include "zzzz_modular_occulus\code\game\Occulus Lore Fixes.dm"

```

Comment:
ikky how the fuck did you put spaces in a file name like this

why does it work

Reply:
i just saw this
what the fuck
HOW DID YOU DO THIS?? @Ikmachek